constraints.txt

```diff
@@ -4,14 +4,18 @@
 #
 #    ./scripts/update_dependencies.sh
 #
-
+-e file:///Users/alecmitchell/Development/bundles/ucb_wallace/Dallinger
```

Comment:
I think you need to have Dallinger installed not in editable mode when generating constraints.

Reply:
It looks like this was an issue with the `update_dependencides.sh` script where the `sed` command which was intended to remove that line (along with all other sed commands) was not working as intended on MacOS. I've fixed the script and re-run it.